src/modules/active-students/styles/list.scss

```diff
@@ -30,12 +31,48 @@
 		margin-top: 40px;
 		border-collapse: separate;
 		border-spacing: 0px 2px;
-		
+
 		td {
 			padding-top: 10px;
 			padding-left: 15px;
 			color: grey;
 			background-color: whitesmoke;
 		}
+		@include media-breakpoint-down(xs) {
+			margin-top: 0;
+			thead{display:none;}
+			tr{
+				display:block;
+			}
+			tr:not(:last-child){
+				border-bottom: 1px solid #ddd;
+			}
+			td{
+				padding-left: .25em;	
+				background-color: transparent;
+			}
+			.picturewrap, .firstname, .lastname{
+				display: inline-block;
+			}
+			.seen,.page{
+				display:block;
+				padding: 0 0 0 58px;
```

Comment:
This is just my personal preference, but I'd rather have these aligned with the name above